### PR TITLE
WIP Add the latest stable PHP version to the Travis testing

### PR DESCRIPTION
**Pull request by mvorisek**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add the latest stable PHP version to the Travis testing
| Type?         | improvement
| Category?     | CO
| BC breaks?    | yno
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | Travis CI should run

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18189)

### CODE REVIEW

File: tools/build/pre-installer-diagram-src.xml [hunk 1]
```diff
@@ -1 +1 @@
...L8nNHznWmHOe4PU1Ukvo4uj6PWTCdNATsDSnOFvq4Vp1mriXokPDfph+jHf9PdElM53iLfTesbRuIehmkLEXYUHrlRRmt2BUy6rpexOMl391JOleR55JoA9WAYtv/+UCst+pzyMS4Sp4mCrpv9Ye5IwJkhq2CT1BOIGHKeJd3feIIQ1ZgvcqOWQe4+Q1NW6WpZf9MHoHmGFLWEGU/mrah65j0Kr3HvG6CuQyUKwgtwaaw9Sc8g3ppGINBeRa4BxZmzxHEICGnYvk5itHQGAK/bFa4ns8jaM65yKyUR90OzQHnnz0fgoO7BGgbM7ivMCL4kP0D0A3CrehpbCBQGwNBBg14Tob0/lmGJpPCouKtY6zahvNrUWUnJsezna034D5KJFgD4WXvi9QBFbYXHJGNDTtm30qQE03Rl11ggq/WzgGEDHGhUURr/eZFVoDG40Ngk+eeYDyBZd5gQywOoRBXNU4erDDC53gsHBVEiq+564CyC7JDDmygxEu5Z0xwlPZs0amiZtuWnIrxR9/peXqAqsgbZ1I7OLtUGgQs1EGOB2dkOdRkZ5onZ9oq6JT2q8b+6ZollVyOv9xdy6Fsx7qT0VZJuN1vv3R5hP3sUkHRdZZzWkCKmwgbcWsIjDoDbrA4xumM3qR8KxbtL66qRqCI3KxuZuUxV5Y0BtZkvKxN5yUJvAILOwYGnyhegQ8pBmdvqUOVsUiVccOhqYh/fOPO1Cjr5ZMhNqXyJe5CM1pr+E596ijwN6izsPB32ngPyKHrGWpxO/f7C9xpuS20j2W4VJEaWXOW3UW3eUGfcosXZqS8o6bwbH56N/PzDq7Mx3l85Ls3ajJ7YW8ELXPLrWjzvgkD8N9YzfXP9VLPtXwvr+GrjzAHlFYLvf9zd/P/sfz+TjUm6xr7/a0W1ozZyxGeBQo2yu0KLmW//37v///3ZY4rIa4edSJZD8y7rOpMHa3Q3insNKq37Gi2lxsO/XiV40la7P8/PM55M/9g5WEVZsKFZ8EUJv2+pCCnCTG4HjUZH+eYLTMkIv+HPD2cI/imsF3S//Rk29vRB9Nggj3xBaNgyO2xXaJA+xKAmo1mwgyuH37/cb2jzey9V9Pu38Jxtl4220G62cSMo/vU37aT6rIv+cF3NXc1i433NLALHFoV2Y5tmb9edE0Q9N8y1u+jdL59WLjo/ADfiM5Iz4DqxXhhY1p
```
Comment (Progi1984) [inline tools/build/pre-installer-diagram-src.xml:1]: Why do you change this file ?

Comment (mvorisek) [reply tools/build/pre-installer-diagram-src.xml:1]: @Progi1984 Please ignore this PR, I am trying to figure out how to make PHP 7.4 installable here and shortly I will close this PR.

Comment (Progi1984) [reply tools/build/pre-installer-diagram-src.xml:1]: @mvorisek Please update the title and put in "WIP" ;)

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
There is no reason to add this on 1.7.7.x, please target the develop branch.

**Pull request closed without merging**